python/model_gen/generate_app_models.py

```diff
@@ -81,60 +86,155 @@ def gen_model(name, model, model_file, version=1, featurize=True):
             h = sha256(f.read()).digest()
             base64_hash = b64encode(h)
             print 'h', base64_hash
-    model_file.write(
+    model_file.write(indent(
         scala_template % {
+            "license": license,
             "name": name,
             "height": model.inputShape()[0],
             "width": model.inputShape()[1],
             "version": version,
-            "base64": base64_hash})
+            "base64": base64_hash},2))
```

Comment:
space after ,

Reply:
i think this might need to be 1 instead of 2

Reply:
I think 2 is correct, it's the number of spaces for the indent, not a number of indent levels.

Reply:
Mm not sure what's going on but the generated file's indent doesn't look right:
```
/**
  * What's in Models.scala
  */
```
```
/**
 * Expected
 */
```